packages/python/libsql-client/libsql_client/http_driver.py

```diff
@@ -1,26 +1,56 @@
 from typing import Any, Dict, List, Optional
+from urllib.parse import urljoin
 import aiohttp
 import base64
 import json
 
 from .driver import _Driver, _RawStmt
-from .errors import ClientResponseError, ClientHttpError
+from .errors import ClientResponseError, ClientHttpError, ClientError
 from .result import ResultSet, Row, Value
 
 class _HttpDriver(_Driver):
     _session: aiohttp.ClientSession
     _url: str
+    _version: Optional[tuple[int, int, int]]
```

Comment:
As a convention, the package uses types from `typing` (i.e. `Tuple`), instead of the builtin types (i.e. `tuple`), to retain compatibility with Python versions before 3.9. (Not that this compatibility is tested, though...)

Reply:
thanks